docs/android/readme_android.md

```diff
@@ -38,8 +38,9 @@ You can follow the standard <a href="/readme.md#importing-the-plugin">Importing
 
 You can follow the standard <a href="/readme.md#samples">Samples</a> process.   
 Please note the details in the <a href="#running-the-samples">Running the samples</a> section when running the samples from a build for Android.  
-> :heavy_exclamation_mark: The EOS Overlay is not implemented yet.   
```

Comment:
I think it is because I started to make changes, then realized that this was a README doc, and I was going to focus on changes to these in a subsequent PR. Ultimately, they will all be replaced.